packages/form-plugin/tests/form.plugin.spec.ts

```diff
@@ -236,7 +236,173 @@ describe('NgxsFormPlugin', () => {
       expect(input.value).toBe('Buy some coffee');
     });
 
-    it('should update the state if "ngxsFormClearOnDestroy" option is provided', () => {
+    it('should update the state if ngxsFormClearOnDestroy option is provided', () => {
```

Comment:
It might be better to add a test with listening to `ofActionDispatched(UpdateForm)` since this is the only action that is done in `ngOnDestroy` when `ngxsFormClearOnDestroy` is truthy. And also another test that ensures that this action is not dispatched. IMHO tests with assuring `store.snapshot` is not intuitive.

Reply:
I think that in this case using the `selectSnapshot` to check the state is the right way to go because the resultant state is the observable behavior of this plugin from a user's perspective.
Testing for an action is potentially testing the internals too much.

Reply:
hmmm it will cover more if we watch ofActionDispatched(UpdateForm) good point. let me change

Reply:
![image](https://user-images.githubusercontent.com/29809885/92462226-a49f6080-f1e3-11ea-84ee-637b6fe3873e.png)
@arturovt is it ok?